{"uuid": "647d1f9e-218d-4a34-a2d8-797b7b80dd97", "children": ["048510c1-5635-42ff-a0ea-d640ae3dd596"], "befores": [{"name": "read_yaml", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\config\\\\oxurl.yaml'\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"E:\\work_pytest\\conftest.py\", line 12, in read_yaml\n    with open(path +r'\\oxurl.yaml', 'r', encoding='utf-8') as file:\n"}, "start": 1589787093934, "stop": 1589787093934}], "start": 1589787093934, "stop": 1589787094244}